strings-csharp/ConvertStringToSpan/ConvertStringToSpan/StringExample.cs

```diff
@@ -0,0 +1,59 @@
+﻿using BenchmarkDotNet.Attributes;
+using System.Runtime.InteropServices;
+using BenchmarkDotNet.Order;
+
+namespace ConvertStringToSpan
+{
+    [Orderer(SummaryOrderPolicy.FastestToSlowest)]
+    [MemoryDiagnoser]
+    public class StringExample
+    {
+        private string _myString = "Hello, World!";
+
+        [Benchmark]
+        public Span<char> ConvertStringToSpanUsingMemoryMarshal()
+        {
+            var charArray = _myString.ToCharArray();
+            var memory = new Memory<char>(charArray);
+            var span = MemoryMarshal.Cast<char, char>(memory.Span);
+
+            return span;
+        }
+
+        [Benchmark]
+        public unsafe Span<char> ConvertStringToSpanUsingUnsafe()
+        {
+            unsafe
+            {
+                fixed (char* ptr = _myString)
+                {
+                    return new Span<char>(ptr, _myString.Length);
```

Comment:
I know this is code from an already published article that somehow bled into this PR, but this code is very dangerous. The Span is targeting heap memory that has been fixed, after the return the fixed block has ended and the target heap object could be moved and now, in theory the Span could be pointing to memory that no longer contains the target string.